ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -319,7 +319,7 @@ ORK_CLASS_AVAILABLE
  
  Note that the `detailText` property of each choice is ignored. Be sure to create localized text for each choice that is short enough to fit in a `UIPickerView` object.
  */
-@property (readonly, copy) NSArray *textChoices;
+@property (readonly, strong) NSArray *textChoices;
```

Comment:
This is intentionally copy, although it's irrelevant as API because it's a readonly property. Do not change.